streamers.json

```diff
@@ -11,7 +11,8 @@
     "channel": "https://www.twitch.tv/avh4",
     "speaking": ["EN"],
     "languages": ["Elm", "Haskell"],
-    "schedule": "[Various times Tu/Th/Su](https://twitter.com/avh4/status/1333478708934369282)"
+    "schedule":
```

Comment:
Are these coming from an auto-formatter? I would rather these changes not be made

Reply:
Yes, sorry about that!